Mathlib/MeasureTheory/MeasurableSpace/Basic.lean

```diff
@@ -1108,6 +1109,26 @@ namespace MeasurableSpace
   rintro _ ⟨u, -, rfl⟩
   exact (show MeasurableSet s from GenerateMeasurable.basic _ <| mem_singleton s).mem trivial
 
+theorem measurableSet_singleton_iff {s t : Set α} :
```

Comment:
Please mention `generateFrom` in the name. Otherwise it looks like the lemma is about `MeasurableSet {a}`.

Reply:
`measurableSet_generateFrom_singleton_iff`? That's quite the mouthful.

Reply:
I updated the theorem name.